docs/UserGuide.md

```diff
@@ -156,37 +182,85 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+RC4HDB data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+RC4HDB data are saved as a JSON file `[JAR file location]/data/RC4data.json`. Advanced users are welcome to update data directly by editing that data file.
+
+:information_source: Do take note that this is not the recommended method to edit data.<br>
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, RC4HDB will discard all data and start with an empty data file at the next run.
 </div>
 
+### Importing from CSV file : `import`
+
+RC4HDB has the ability to import data through .csv files. In order for RC4HDB to find your files, place them in the data directory, `[JAR file location]/data`.
+
+Format: `import FILENAME`
+
+:information_source: The csv file that you want to have imported must follow this [format](#csv-file-format).<br>
+
+Examples:
+* `import students.csv`
+* `import residents.csv`
+
+### Exporting to CSV file : `export`
+
+RC4HDB has the ability to export data to .csv files. The file will be safe to remove from the directory, `[JAR file location]/data`.
+
+Format: `export FILENAME`
+
+:information_source: The csv file will be exported in this [format](#csv-file-format).<br>
+
+Examples:
+* `export students.csv` will export the current data file into a csv file named `students.csv`
+
+### CSV file format
+
+| INDEX 	| NAME 	| PHONE NUMBER 	|    EMAIL      | FLOOR 	| UNIT NUMBER 	| TAGS 	|
+|-------	|------	|--------------	| ------        | -------	|-------------	|------	|
+| 1      	| John Doe | 91234567  	| johnDoe@gmail.com       | 05      	|  08         	| Male 	|
+| 2      	| Maggie Smith	| 98765432	| maggieSmith@gmail.com        | 04   	|  01         	| Female|
+
```

Comment:
Update the table to include
1. `INDEX`
2. `NAME`
3. `PHONE_NUMBER`
4. `EMAIL`
5. `FLOOR`
6. `UNIT`
7. `GENDER`
8. `HOUSE`
9. `MATRIC_NO`
10. `MAJOR`
11. `TAGS`